### PR TITLE
Makefile: add BEAT_ variables

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,12 @@
-BEATNAME=gpfsbeat
-BEAT_DIR=github.com/itkovian/gpfsbeat
+BEAT_NAME=gpfsbeat
+BEAT_PATH=github.com/itkovian/gpfsbeat
+BEAT_DESCRIPTION=gpfsbeat is an Elastic Beat that parses GPFS mmrepquota e.a. and sends them to ELK.
+BEAT_URL=https://github.com/itkovian/gpfsbeat
+BEAT_DOC_URL=https://github.com/itkovian/gpfsbeat
```
Comment (boegel) [inline Makefile]: hpcugent?

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,12 @@
-BEATNAME=gpfsbeat
-BEAT_DIR=github.com/itkovian/gpfsbeat
+BEAT_NAME=gpfsbeat
+BEAT_PATH=github.com/itkovian/gpfsbeat
```
Comment (boegel) [inline Makefile]: hpcugent?

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,12 @@
-BEATNAME=gpfsbeat
-BEAT_DIR=github.com/itkovian/gpfsbeat
```
Comment (boegel) [inline Makefile:2]: these are not used anywhere?

Comment (stdweird) [reply Makefile:2]: have no clue; i was able to build iobeat like this

Comment (stdweird) [reply Makefile:2]: i grepped, nothing found with the old variables

Comment (stdweird) [reply Makefile:2]:
```
[stdweird@regice beats]$ git grep BEATNAME
generator/beat/CHANGELOG.md:- Renamed BEATNAME to BEAT_NAME.
[stdweird@regice beats]$ git grep BEAT_DIR
generator/beat/CHANGELOG.md:- Renamed BEAT_DIR to BEAT_PATH.
generator/metricbeat/CHANGELOG.md:* Renamed BEAT_DIR to BEAT_PATH.
```

**Pull request merged**